src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerGetYourAudioIntTest.java

```diff
@@ -0,0 +1,214 @@
+            []
+            """;
+        JSONAssert.assertEquals(expectedJson, actualJson, JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    void getYourAudioExpiredShouldReturnEmptyArrayInResponseBodyWhenNoExpiredMediaRequestExists() throws Exception {
+
+        var requestor = dartsDatabase.createIntegrationTestUserAccountEntity(systemUser);
+        dartsDatabase.createAndLoadCurrentMediaRequestEntity(requestor);
+
+        var requestBuilder = get(URI.create(String.format("/audio-requests?expired=%s", TRUE)))
+            .header(
+                "user_id",
+                requestor.getId()
+            );
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder)
+            .andExpect(status().isOk())
+            .andReturn();
+
+        String actualJson = mvcResult.getResponse().getContentAsString();
+        String expectedJson = """
+            []
+            """;
+        JSONAssert.assertEquals(expectedJson, actualJson, JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    void getYourAudioShouldReturnBadRequestWhenExpiredQueryParamIsNotPresent() throws Exception {
+
+        var requestor = dartsDatabase.createIntegrationTestUserAccountEntity(systemUser);
+
+        var requestBuilder = get(URI.create("/audio-requests"))
+            .header(
+                "user_id",
+                requestor.getId()
+            );
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder)
+            .andExpect(status().isBadRequest())
+            .andReturn();
+
+        String actualJson = mvcResult.getResponse().getContentAsString();
+        String expectedJson = """
+            {
+                "type": "about:blank",
+                "title": "Bad Request",
+                "status": 400,
+                "detail": "Required parameter 'expired' is not present.",
+                "instance": "/audio-requests"
+            }
+            """;
+        JSONAssert.assertEquals(expectedJson, actualJson, JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    void getYourAudioShouldReturnBadRequestWhenUserIdHeaderIsNotPresent() throws Exception {
+
+        var requestBuilder = get(URI.create(String.format("/audio-requests?expired=%s", FALSE)));
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder)
+            .andExpect(status().isBadRequest())
+            .andReturn();
+
+        String actualJson = mvcResult.getResponse().getContentAsString();
+        String expectedJson = """
+            {
+                "type": "about:blank",
+                "title": "Bad Request",
+                "status": 400,
+                "detail": "Required header 'user_id' is not present.",
+                "instance": "/audio-requests"
+            }
+            """;
+        JSONAssert.assertEquals(expectedJson, actualJson, JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    void getYourAudioShouldReturnEmptyArrayInResponseBodyWhenNoWhenNoUserAccountExists() throws Exception {
```

Comment:
in function name getYourAudioShouldReturnEmptyArrayInResponseBodyWhenNoWhenNoUserAccountExists pepeated WhenNo